Gemfile

```diff
@@ -5,7 +5,9 @@ gem 'rails', '4.1.5'
 
 gem 'actionpack-action_caching'
 gem 'coffee-rails'
-gem 'crowdblog', '0.5.0'
+gem 'crowdblog',
```

Comment:
Not sure if this is what we want in the long term: we can release that crowdblog version to ruby gems and get it from them instead of pointing to a specific tag in out git repo